### PR TITLE
Added option to print to standard output

### PR DESCRIPTION
**Pull request by rsateler**
This commit adds the ability to print to standard output instead to a log file. Using:
`logkeys --start -o -`

### CODE REVIEW

File: src/logkeys.cc [hunk 1]
```diff
@@ -423,7 +428,11 @@ int main(int argc, char **argv)
   int noclose = 1;  // don't close streams (stderr used)
   if (daemon(nochdir, noclose) == -1)  // become daemon
     error(EXIT_FAILURE, errno, "Failed to become daemon");
-  close(STDIN_FILENO); close(STDOUT_FILENO);  // leave stderr open
+  close(STDIN_FILENO);
+  // leave stderr open
+  if (args.logfile.compare("-") != 0) {
```
Comment (kernc) [inline src/logkeys.cc]: Doesn't `std::string` overload `operator==`? Wouldn't `args.logfile == "-"` be clearer?

**Comment by rsateler** [comment]
Changed code to use operator `==`

File: src/logkeys.cc [hunk 2]
```diff
@@ -393,6 +393,11 @@ int main(int argc, char **argv)
   if (!args.keymap.empty() && (!(args.flags & FLAG_EXPORT_KEYMAP) && args.us_keymap)) {  // exporting uses args.keymap also
     error(EXIT_FAILURE, 0, "Incompatible flags '-m' and '-u'. See usage.");
   }
+
+  // check for incompatible flags: if posting remote and output is set to stdout
+  if (args.post_size != 0 && ( args.logfile.compare("-") == 0 )) {
```
Comment (kernc) [inline src/logkeys.cc]: Sorry, could you apply that here too? :blush:

**Comment by rsateler** [comment]
Sorry, done!

**Comment by kernc** [comment]
Great, thanks!

**Pull request merged**